### PR TITLE
Add Case 9 For Simple Interest

### PR DESCRIPTION
**Pull request by Er1ckkkk**

### CODE REVIEW

File: src/com/paynemiller/Runner.java [hunk 1]
```diff
@@ -57,6 +56,17 @@ private static void startCalculator() {
...
   }
 
+  private static void startCharacterCounter() {
+    System.out.println("Enter a word!");
+    String word = keyboard.nextLine();
+    CharacterCounter counter = new CharacterCounter();
+    try {
+      counter.countCharacters(word);
+    } catch (EmptyWordException e) {
+      e.printStackTrace(); // TODO: Gracefully handle exception.
```
Comment (paynemiller92) [inline src/com/paynemiller/Runner.java:66]: Has this `TODO` been handled?

File: src/com/paynemiller/ch3/CompoundInterestComputer.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        interestRate = interestRate / 100;
+        compoundInterest = Math.pow(principalAmount * (1 + (interestRate / compoundedPerYear)), compoundedPerYear* numYears);
+    }
+
+    public void showCompoundInterest(){
+        System.out.print("$" + principalAmount + "invested at " + interestRate + "% for " + numYears + " years" +
+                "%n compounded " + compoundedPerYear + " times per year is $" + compoundInterest);
+    }
+
+    public void output(){
```
Comment (paynemiller92) [inline src/com/paynemiller/ch3/CompoundInterestComputer.java:35]: Is there a more descriptive method name besides `output()`? Think about what this function does.

File: src/com/paynemiller/ch3/SimpleInterestComputer.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        rateOfInterest = rateOfInterest / 100;
+        simpleInterest = principalAmount * (1 + (rateOfInterest * numYears));
+        simpleInterest = Math.round(simpleInterest * 100.0) / 100.0; // rounds to the next decimal
+    }
+
+    public void showSimpleInterest(){
+        System.out.print("After " + numYears + " years at " + rateOfInterest * 100 + "%, the investment will be worth $" + simpleInterest);
+    }
+
+    public void output(){
```
Comment (paynemiller92) [inline src/com/paynemiller/ch3/SimpleInterestComputer.java:31]: Is there a more descriptive name besides `output()`? What does this method do?

File: src/com/paynemiller/ch3/SimpleInterestComputer.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        promptForInput();
+        calculateSimpleInterest();
+        showSimpleInterest();
+    }
+
+    //formatting could be fixed
```
Comment (paynemiller92) [inline src/com/paynemiller/ch3/SimpleInterestComputer.java:37]: IntelliJ [can auto-format](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html) it for you!

**Review by paynemiller92 — COMMENTED** [review]
Just a few minor comments!

File: src/com/paynemiller/ch3/SimpleInterestComputer.java (hunk 4 above)
Comment (Er1ckkkk) [reply src/com/paynemiller/ch3/SimpleInterestComputer.java]: Fixed the formatting issues.

https://github.com/paynemiller92/programming-practice-erick/pull/12/commits/d45185acb0be4e3b0c4c364bdb8f3edc78e79b20

File: src/com/paynemiller/ch3/CompoundInterestComputer.java (hunk 2 above)
Comment (Er1ckkkk) [reply src/com/paynemiller/ch3/CompoundInterestComputer.java]: Changed name of the method.

https://github.com/paynemiller92/programming-practice-erick/pull/12/commits/cc1af3f74b229c1f93959aad8523c7da984a7078

File: src/com/paynemiller/ch3/SimpleInterestComputer.java (hunk 3 above)
Comment (Er1ckkkk) [reply src/com/paynemiller/ch3/SimpleInterestComputer.java]: Changed name of the method.

https://github.com/paynemiller92/programming-practice-erick/pull/12/commits/cc1af3f74b229c1f93959aad8523c7da984a7078

**Comment by paynemiller92** [comment]
Great work @Er1ckkkk ! 🚀

**Pull request merged**